src/screens/NewFlowSensorScreen.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import type { EntityID, FlowSensorMutator } from 'brewskey.js-api';
+import type { Navigation } from '../types';
+
+import * as React from 'react';
+import DAOApi from 'brewskey.js-api';
+import { FlowSensorStore, waitForLoaded } from '../stores/DAOStores';
+import InjectedComponent from '../common/InjectedComponent';
+import Container from '../common/Container';
+import Header from '../common/Header';
+import FlowSensorForm from '../components/FlowSensorForm';
+import flatNavigationParamsAndScreenProps from '../common/flatNavigationParamsAndScreenProps';
+
+type InjectedProps = {|
+  navigation: Navigation,
+  tapId: EntityID,
+|};
+
+@flatNavigationParamsAndScreenProps
+class NewFlowSensorScreen extends InjectedComponent<InjectedProps> {
+  _onFormSubmit = async (values: FlowSensorMutator): Promise<void> => {
+    const { navigation, tapId } = this.injectedProps;
+    const clientID = DAOApi.FlowSensorDAO.post(values);
+    // todo promise never resolves for some reason
+    // but it should like in all other similar cases
+    await waitForLoaded(() => FlowSensorStore.getByID(clientID));
```

Comment:
this is a really weird bug. the promise never resolves, despite the fact that it works absolutely okey in all other similar cases. It may be another environment bug or maybe I don't see obvious mistake and just need to sleep :/

Reply:
It probably has to do with the API bug